plugins/statistics.koplugin/main.lua

```diff
@@ -2687,6 +2688,43 @@ function ReaderStatistics:getReadBookByDay(month)
     return per_day
 end
 
+function ReaderStatistics:getReadingDurationBySecond(date)
+    local sql_stmt = [[
+        SELECT
+            start_time - strftime('%s', ?, 'utc') as start,
+            start_time - strftime('%s', ?, 'utc') + duration as end,
+            id_book book_id,
+            book.title book_title
+        FROM   page_stat_data
+        JOIN   book ON book.id = page_stat_data.id_book
+        WHERE  start_time BETWEEN strftime('%s', ?, 'utc')
+                              AND strftime('%s', ?, 'utc', '+24 hours', '-1 second')
```

Comment:
Any idea why we/I used 'utc' in similar queries ? There must be a good reason, but I don't remember/see why :)
May be because we insert without more thinking needed as utc times, and it's just fine on our devices. But we sometimes use 'localtime' when formatting - all this may just work and is right - just asking if you wondered and give it some thoughts, and if that makes sense to you :)

Reply:
I copied them from other queries :) Guess it's because the timestamps we get from `os.time()` and inserted to db are utc-based? Here the `?`s are bound with local time strings like '2022-11-11', so it has to be converted to utc seconds?

Reply:
Turns out I can just use the timestamp instead of date strings. Changed to use timestamp.